src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs

```diff
@@ -76,6 +72,20 @@ public IEnumerable<ApplicationContext> InstanceDaemonApps(IServiceProvider servi
                     throw new NetDaemonException($"Error instance the app from the file {file}", e);
                 }
             }
+
+            var appTypesWithoutYamlConfig = _loadedDaemonApps
+                    .Where(appType => !result.Select(x => x.ApplicationInstance.GetType()).Contains(appType));
+            
+            foreach (var appType in appTypesWithoutYamlConfig)
+            {
+                var appId = appType
+                        .GetCustomAttributes(typeof(NetDaemonAppAttribute), true)
+                        .Cast<NetDaemonAppAttribute>()
+                        .FirstOrDefault()?.Id ?? appType.Name;
```

Comment:
@FrankBakkerNl , is this viable?
```suggestion
                var appId = appType.GetCustomAttribute<NetDaemonAppAttribute>()?.Id ?? appType.Name;
```

Reply:
Yea, that is what I ment it is basically the same as you had before, just en bit shorter and eaaier to read

Reply:
it sure is!